tailormap-components/projects/core/src/lib/feature-form/form-field/formfield.component.html

```diff
@@ -29,3 +30,17 @@
     </mat-select>
   </div>
 </div>
+
+<div [formGroup]="groep" class="form-field" *ngIf="editing && isCheckboxAttribute(attribute)">
+  <div class="label">{{attribute.label}}</div>
+  <mat-checkbox  [checked]="checkboxValue()" (change)="onCheckboxChange($event)"></mat-checkbox>
+</div>
+
+<div [formGroup]="groep" class="form-field" *ngIf="editing && isDateAttribute(attribute)">
+  <div class="label">{{attribute.label}}</div>
+  <mat-form-field>
+    <input matInput [matDatepicker]="picker" [value]="getDate()" (dateChange)="changeDate($event)" [placeholder]="getDate()">
+    <mat-datepicker-toggle matSuffix [for]="picker"></mat-datepicker-toggle>
+    <mat-datepicker [startAt]="getDate()" #picker></mat-datepicker>
+  </mat-form-field>
+</div>
```

Comment:
Might be better to move to separate components in user-interface/generic-components. That way we can style form components generically according to the concept7 style